pkg/manager/setconfig.go

```diff
@@ -86,16 +86,16 @@ func (m *Manager) SetNetworkConfig(configName store.ConfigName, updates map[stri
 	}
 
 	if m.ChangeStore.Store[store.B64(configChange.ID)] != nil {
-		log.Println("Change ID", store.B64(configChange.ID), "already exists - not overwriting")
+		log.Info("Change ID", store.B64(configChange.ID), "already exists - not overwriting")
```

Comment:
warning ?

Reply:
It's not really a warning - more of a no-op